ngi_pipeline/utils/communication.py

```diff
@@ -13,9 +14,33 @@ def mail(recipient, subject, text, origin="ngi_pipeline"):
     s.sendmail('funk_002@nestor1.uppmax.uu.se', recipient, msg.as_string()) 
     s.quit()
 
+@with_ngi_config
+def mail_ngi_log_message(se, log_level, message, config=None, config_file_path=None ):
+    """Wrapper around mail_analysis reporting log messages by mail. 
+    It is reading recipient from the config object, and the project characteristics from the se a.k.a. "sample environment" object. 
+    See SampleEnv for details about sample environment.
+    """
+    # we are expecting the mail recipient to be configured, but have to default to 
+    # "ngi_pipeline_operators@scilifelab.se" if it is missing
+    # Actually I would be happier have it as a compulsory config item, since we are
+    # planning deploy this to other places where this email is not available or not appropiate
+    try:
+        recipient = config.get('mail')['recipient']
+    except TypeError:
+        recipient = "ngi_pipeline_operators@scilifelab.se"
+
+    mail_analysis(  project_name = se.project_name,
+                    sample_name = se.sample_id,
+                    engine_name = se.engine,
+                    level = log_level,
+                    info_text = message,
+                    workflow = se.workflow,
+                    recipient = recipient) 
+
+
 def mail_analysis(project_name, sample_name=None, engine_name=None,
                   level="ERROR", info_text=None, workflow=None,
-                  recipient="ngi_pipeline_operators@scilifelab.se",
+                  recipient="szilveszter.juhos@scilifelab.se",
```

Comment:
Yep, though in a way it is used now it is always defaulting to ngi_pipeline_operators. Fixed.